team-arrival-checklist/form.tex

```diff
@@ -0,0 +1,38 @@
+\documentclass[a4paper, 12pt]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage[default,osfigures]{opensans}
+
+\usepackage[margin=0.5in]{geometry}
+\usepackage{tabularx}
+\setlength{\parindent}{0cm}
+
+\newcolumntype{Y}{>{\centering\arraybackslash}X}
+
+\begin{document}
+
+\begin{center}
+
+    {\huge{Team Arrival Checklist}}
+
+    \medskip
+
+\end{center}
+
+When a team arrives on Saturday morning, please check them off on this sheet by filling in their time of arrival. If the sheet indicates that they have special requirements, consult the 'Team Arrival Special Kit Requirements' sheet.
+
+\begin{center}
+\begin{table}[h]
+\large
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|}
```

Comment:
This table doesn't have a top line, not sure if that's intentional.

Reply:
It does, there's a `\hline` after the `\begin{tabularx}` command.

Reply:
There's an `\hline` after the first (headings) row, but not one before it. This means that the top of the header row is missing a line:

![2019-03-24-223213_903x255_scrot](https://user-images.githubusercontent.com/336212/54886866-bd4b7780-4e84-11e9-8ead-1bbb21bbae0a.png)